templates/default/nginx.yaml.erb

```diff
@@ -1,6 +1,12 @@
 instances:
 <% @instances.each do |i| -%>
   - nginx_status_url: <%= i['nginx_status_url'] %>
+    <%= "ssl_validation: #{i['ssl_validation']}" unless i['ssl_validation'].nil? -%>
```

Comment:
@olivielpeau I've pushed a5e1f25 with the recommended changes; I also added spec tests and updated the examples in the nginx recipe to demonstrate the new features.